decidim-admin/spec/system/admin_manages_organization_areas_spec.rb

```diff
@@ -75,16 +75,36 @@
       end
 
       it "can delete them" do
-        within find("tr", text: translated(area.name)) do
-          accept_confirm { click_link "Delete" }
-        end
+        click_delete_area
 
         expect(page).to have_admin_callout("successfully")
 
         within ".card-section" do
           expect(page).to have_no_content(translated(area.name))
         end
       end
+
+      context "when a participatory space associated to a given area exist" do
+        let!(:process) { create(:participatory_process, organization: area.organization, area: area) }
+
+        it "can not be deleted" do
+          click_delete_area
+          expect(area.reload.destroyed?).to be false
+          expect(page).to have_admin_callout("Area has depedent spaces")
+        end
+      end
+    end
+  end
+
+  #---------------------------------------------------
```

Comment:
Please remove this.

Reply:
it is not breaking anything and it makes navigation in the artifacts much more easy.

Reply:
Please, this is an already solved discussion. Can you stick to what we agreed in the past? 🙏